mlflow/server/js/src/components/modals/RenameRunFormView.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+import { Formik, Field } from 'formik';
+
+import { validationSchema } from './validation';
+
+/**
+ * Component that renders a form for updating a run's name. Expects to be 'closeable'
+ * (i.e. rendered within a closeable dialog) and so accepts an `onClose` callback.
+ */
+class RenameRunFormView extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    onClose: PropTypes.func.isRequired,
+    runName: PropTypes.string.isRequired,
+    experimentId: PropTypes.number.isRequired
+  }
+
+  renderForm = (renderProps) => {
+    const {
+      handleSubmit,
+      isSubmitting,
+    } = renderProps;
+    const saveText = isSubmitting ? "Saving..." : "Save";
+    return <form onSubmit={handleSubmit} style={styles.form}>
+      <div style={styles.formField}><h2 style={{"marginTop": "0px"}}>Rename Run</h2></div>
+      <div style={styles.label}>New run name:</div>
```

Comment:
You should put the label inside the Field. You could use bootstrap styles as well. Something like this (had this lying around):
```
import { FormGroup, ControlLabel, FormControl, HelpBlock } from 'react-bootstrap';

export default function TextField(
  { field, form: { touched, errors }, label, ...props },
) {
  const error = errors[field.name];
  return <FormGroup
    controlId={field.name}
    validationState={touched[field.name] && error ? 'error' : null}
  >
    <ControlLabel>{label}</ControlLabel>
    <FormControl
      type="text"
      {...field}
    />
    { touched[field.name] && error ? <HelpBlock>{error}</HelpBlock> : null }
  </FormGroup>
}
```